File: bffs/web-bff/src/routes/adminRoutes.js
const express = require("express");
const { getBreakerSnapshot } = require("../utils/circuitBreaker");

const router = express.Router();

router.get("/breakers", (_req, res) => {
  res.json({
    service: "web-bff",
    breakers: getBreakerSnapshot()
  });
});

module.exports = router;

File: bffs/mobile-bff/app/routers/admin_routes.py
from fastapi import APIRouter
from app.utils.circuit_breaker import breaker_snapshot

router = APIRouter(prefix="/admin", tags=["Admin"])


@router.get("/breakers")
async def breakers():
    return {
        "service": "mobile-bff",
        "breakers": breaker_snapshot()
    }